packages/user-interface/src/components/Heading.tsx

```diff
@@ -6,7 +6,7 @@ import {
 } from "@gemeente-denhaag/typography";
 import classNames from "classnames";
 
-type Headers = "h1" | "h2" | "h3" | "h4" | "h5";
+export type Headers = "h1" | "h2" | "h3" | "h4" | "h5";
```

Comment:
Wilde je deze gebruiken in de extra details?

Reply:
Ja ook i.v.m. de nesting, maar dit heb ik dus weg gelaten, dus ik haal dit nu ook weg uit de code.